src/huggingface_hub/inference/_client.py

```diff
@@ -157,6 +157,7 @@ def __init__(
         timeout: Optional[float] = None,
         headers: Optional[Dict[str, str]] = None,
         cookies: Optional[Dict[str, str]] = None,
+        proxy: Optional[str] = None,
```

Comment:
Could you name it `proxies` to be closer to `requests`/`httpx` naming? (even though `aiohttp` uses `proxy`). Also type annotate it as `Optional[Any]` since it can accept different type (string or dict[str, str]) and we don't want to be opinionated about it. 

```suggestion
        proxies: Optional[Any] = None,
```

Reply:
I just pushed the changes you suggested ! Is everything good for you ?